main.go

```diff
@@ -540,24 +540,17 @@ func main() {
 		cluster.SslOpts = sslOpts
 	}
 
-	// Use separate session to create the keyspace which we will use in the main session
-	prepare_session, err := cluster.CreateSession()
+	session, err := cluster.CreateSession()
 	if err != nil {
 		log.Fatal(err)
 	}
-	PrepareDatabase(prepare_session, replicationFactor)
-	prepare_session.Close()
+	defer session.Close()
 
+	PrepareDatabase(session, replicationFactor)
 	// NOTE: set the keyspace explicitly to workaround the following bug:
 	//       https://github.com/gocql/gocql/issues/1621
 	cluster.Keyspace = keyspaceName
```

Comment:
Looks like the https://github.com/gocql/gocql/issues/1621 bug was fixed and, as I can see, ported to the `scylladb`'s fork.
So, shouldn't this line be removed too?
In this case the second session will become totally redundant even if was compatible with the newest driver.

Reply:
You are right, I is unnecessary after bug fix.